f8a_worker/utils.py

```diff
@@ -608,6 +612,11 @@ def get_gh_contributors(url):
     try:
         response = requests.get("{}?per_page=1".format(url),
                                 headers=get_header())
+        # If status code is 404 or 204 then don't retry
+        if response.status_code == 404:
```

Comment:
If GH URL is not valid it will stop the execution on 572 it self, but in case of a valid GH URL to an empty repo, we dont get 204 for API called for repo details, 
for example this empty repo
https://github.com/olafury/vitrotestui
we call API to get fork/subscriber etc
https://api.github.com/repos/olafury/vitrotestui
It doent give 204 instead you get a response with all fields set to 0.
So we keep another validation while checking for contributors if we get 204 it eans that is an empty repo..